src/cbmc/cbmc_parse_options.cpp

```diff
@@ -909,8 +909,11 @@ bool cbmc_parse_optionst::process_goto_program(
     }
   
     // do partial inlining
-    status() << "Partial Inlining" << eom;
```

Comment:
This should rather be solved (inside goto_inline) by preventing direct function calls in _start to be inlined.